src/sandesh/Assignment_23/CountOfPrimeNumbersWithinGivenRange.java

```diff
@@ -0,0 +1,35 @@
+package sandesh.Assignment_23;
+
+import java.util.Scanner;
+
+public class CountOfPrimeNumbersWithinGivenRange {
+	
+	int getPrimeNumbersCount(int startPoint, int endPoint) {
+		int primeNumbersCount = 0;
+		for(int index=startPoint; index<= endPoint; index++) {
+			boolean isPrime = true;
+			for(int innerIndex=2; innerIndex<index/2 + 1; innerIndex++) {
+				if(index%innerIndex  == 0) {
+					isPrime = false;
+					break;
+				}
+			}
+			if(isPrime && index >=2) {
```

Comment:
only boolean is sufficient to check condition index will be always >=2 as per your code.
Let me know if you have different thinking ion this

Reply:
Hello Harsh, I added '&& index >=2' for checking 0 and 1 are not prime. If user enters 0 or 1, then this should return false as 0 and 1 are not prime.